### PR TITLE
guest_agent_information: Add info about vmi status infoSource

### PR DESCRIPTION
**Pull request by oshoval**
Explain the new field and it's possible values.

Additional:
Fix indentation of status snippets.

**Special notes for your reviewer**:
Related to https://github.com/kubevirt/kubevirt/pull/6793

### CODE REVIEW

**Comment by oshoval** [comment]
/hold
until https://github.com/kubevirt/kubevirt/pull/6793 is merged

**Comment by cwilkers** [comment]
/lgtm
ack on the hold.

**Comment by oshoval** [comment]
/hold cancel

@EdDev @AlonaKaplan 
can you please take a look ?

File: docs/virtual_machines/guest_agent_information.md [hunk 1]
```diff
@@ -40,16 +40,21 @@ GA info with merged into status
...
+              - fe80::858:aff:fef4:17/64
+              mac: 0a:58:0a:f4:00:17
+              name: default
+
+When the Guest Agent is not present in the Virtual Machine, the Guest Agent information is not shown. No error is reported because the Guest Agent is an optional component.
+
+The infoSource field indicates where the info is gather from. Valid values:
+1. domain: the info is based on the domain spec
+2. guest-agent: the info is based on Guest Agent report
+3. domain, guest-agent: the info is based on both the domain spec and the Guest Agent report
```
Comment (EdDev) [inline docs/virtual_machines/guest_agent_information.md:57]: For some reason, this is not ending up well formatted on the UI. Check the preview.

**Comment by oshoval** [comment]
Thanks, fixed the list, it isn't a pure markdown it seems.

Aligned the yaml parts to be with ident like the real one have

File: docs/virtual_machines/guest_agent_information.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ The condition tells that the GA is connected and can be used.
...
-        conditions:
-            - lastProbeTime: "2020-02-28T10:22:59Z"
-            lastTransitionTime: null
-            status: "True"
-            type: AgentConnected
+      conditions:
+      - lastProbeTime: "2020-02-28T10:22:59Z"
+        lastTransitionTime: null
+        status: "True"
+        type: AgentConnected
```
Comment (EdDev) [inline docs/virtual_machines/guest_agent_information.md:20]: This seems an unrelated change.
If you want to reformat things, maybe create a pre-commit for it.

**Comment by oshoval** [comment]
split to commits (had hunch you would ask)

**Comment by EdDev** [comment]
> split to commits (had a hunch you would ask)

Thanks. I see you did other formatting on the other part as well, can you include them in the first commit as well?
I just want to see the added attribute on the second commit. Is this possible?

**Comment by oshoval** [comment]
> Thanks. I see you did other formatting on the other part as well, can you include them in the first commit as well? I just want to see the added attribute on the second commit. Is this possible?

Done

**Review by EdDev — APPROVED** [review]
Thank you!

File: docs/virtual_machines/guest_agent_information.md [hunk 3]
```diff
@@ -40,16 +40,22 @@ GA info with merged into status
...
         ipAddresses:
         - 10.244.0.23/24
         - fe80::858:aff:fef4:17/64
         mac: 0a:58:0a:f4:00:17
         name: default
 
-When the Guest Agent is not present in the Virtual Machine, the information is not shown. No error is reported because the Guest Agent is an optional component.
+When the Guest Agent is not present in the Virtual Machine, the Guest Agent information is not shown. No error is reported because the Guest Agent is an optional component.
 
+The infoSource field indicates where the info is gather from. Valid values:
```
Comment (AlonaKaplan) [inline docs/virtual_machines/guest_agent_information.md:54]: s/gather/gathered

Comment (oshoval) [reply docs/virtual_machines/guest_agent_information.md]: done

**Comment by oshoval** [comment]
addressed comments

**Review by EdDev — APPROVED** [review]
